src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
You can extract Colors.values() result to separate variable and reuse it in both lines:

```suggestion
        Colors[] colorsArray = Colors.values();

        int index = new Random().nextInt(colorsArray.length);
        return colorsArray[index].toString();
```